packages/ui/origin-ui-core/src/components/organization/organizationMenu.ts

```diff
@@ -78,8 +79,9 @@ export const useOrganizationMenu = (): IOrganizationMenuItem[] => {
             key: 'register-irec',
             label: 'Register I-REC',
             component: IRECRegisterForm,
-            feature: OriginFeature.IRec,
             show:
+                config && // sometimes context returns null as config
```

Comment:
I hate that kind of comments, but this is the problem, that is hard to verify, due to problems @alexworker23  mentioned with rerendering.

If you have another idea, please share